cypress/integration/InHidingPage.feature

```diff
@@ -17,13 +17,19 @@ Feature: Are you or your relative's in hiding or in danger page
     When I select "no i'm not worried about searching in public" radio button
     Then I click the "continue" button
     Then I am on the "eligible for both" page
+    Then I should see that both services are eligible
 
-  Scenario: Selecting Yes publicly searching for my family could be dangerous and clicking continue navigates to the Eligible for Both Services page
+  Scenario: Selecting Yes publicly searching for my family could be dangerous and clicking continue navigates to the Eligible for non public page
     When I select "yes publicly search for my family could be dangerous" radio button
     Then I click the "continue" button
-    Then I am on the "eligible for one" page
+    Then I am on the "eligible for one non public" page
+    Then I should see the no public services advice
 
-  Scenario: Selecting Im not sure and clicking continue navigates to the Eligible for Both Services page
+  Scenario: Selecting Im not sure and clicking continue navigates to the Eligible for Both Services page and back
     When I select "i'm not sure" radio button
     Then I click the "continue" button
     Then I am on the "eligible for both" page
+    Then I should see that both services are eligible
+    When I click the back link
```

Comment:
Do we need to test the back action on a page that isn't related to this one?

Reply:
I wasn't sure the best way to test it, as the same template is accessed from 'in-hiding' and 'where-should-we-start-looking' I wanted to check it was returning to the right place.

Reply:
It would be more conventional to have a separate scenario for testing the back link.

Reply:
Thats fine, I don't know which is right, but thought I should ask the question

Reply:
I still think the back link should have it's own scenario ideally, but happy to roll with this for MVP timescale reasons.